README.md

```diff
@@ -1,44 +1,183 @@
+        }, {
+          "name": "zipcode",
+          "type": "esriFieldTypeString",
+          "alias": "zipCode"
+        }, {
+          "name": "streetname",
+          "type": "esriFieldTypeString",
+          "alias": "streetname"
+        }, {
+          "name": "addressnumber",
+          "type": "esriFieldTypeString",
+          "alias": "addressNumber"
+        }, {
+          "name": "guid",
+          "type": "esriFieldTypeString",
+          "alias": "guid"
+        }, {
+          "name": "addresstype",
+          "type": "esriFieldTypeString",
+          "alias": "addressType"
+        }, {
+          "name": "addressplacement",
+          "type": "esriFieldTypeString",
+          "alias": "addressPlacement"
+        }, {
+          "name": "addresssource",
+          "type": "esriFieldTypeString",
+          "alias": "addressSource"
+        }, {
+          "name": "addressauthority",
+          "type": "esriFieldTypeString",
+          "alias": "addressAuthority"
+        }, {
+          "name": "datelastupdated",
+          "type": "esriFieldTypeDate",
+          "alias": "dateLastUpdated"
+        }, {
+          "name": "sourceservice",
+          "type": "esriFieldTypeString",
+          "alias": "sourceService"
+        }, {
+          "name": "aid",
+          "type": "esriFieldTypeDouble",
+          "alias": "aid"
+        }
+      ]
+    },
+    "aggregate_schemas": {}
+  },
+  "aggregate_schemas": {}
+}
+```
+
+### Register a new Source
+
+_Body_
+
+`PUT {server}/composite/{indicator_name}/{source_name}`
+
+
+```json
+{
+      "url": "https://services7.arcgis.com/BBpPn9wZu2D6eTNY/arcgis/rest/services/ORANGE_COUNTY_WITH_FLDS/FeatureServer/0/query",
+      "fieldMap": {
+        "state": "State",
+        "county": "Add_cnty",
+        "incorporatedmunicipality": "MailingCit",
+        "unincorpcounty": "Add_uninc",
+        "postalcommunityname": "FireDist",
+        "zipcode": "ZipCode",
+        "streetname": "Name",
+        "addressnumber": "HouseNum",
+        "guid": "guid",
+        "addresstype": "LBCSDesc",
+        "addressplacement": "Add_plcmnt",
+        "addresssource": "Add_src",
+        "addressauthority": "Add_auth",
+        "datelastupdated": "EditDate"
+    },
+    "extent": {
+        "xmin": -8825271.4547,
+        "ymin": 4280522.407899998,
+        "xmax": -8788727.2223,
+        "ymax": 4334516.740699999,
+        "spatialReference": {
+            "wkid": 102100,
+            "latestWkid": 3857
+        }
+    }
+}
+```
+
+### Query Indicator
 
-- `docker build -t koop-provider-sample .`
-- `docker run -it -p 8080:8080 koop-provider-sample`
+`GET {server}/composite/{indicator_name}/FeatureServer/0/query?returnCountOnly=true`
 
-## Publish to npm
-- run `npm init` and update the fields
-  - Choose a name like `koop-provider-foo`
-- run `npm publish`
+[Example at localhost:3000](localhost:3001/composite/addresses/FeatureServer/0/query?returnCountOnly=true)
```

Comment:
you could link somewhere to the deployed version at: https://adapters.koopernetes.com/compsite though we may want to put an HTML page up there